disputer/disputer.js

```diff
@@ -18,19 +18,21 @@ class Disputer {
     this.web3 = this.empClient.web3;
   }
 
+  // Update the client and gasEstimator clients.
+  // If a client has recently updated then it will do nothing.
+  update = async () => {
+    await this.empClient.update();
+    await this.gasEstimator.update();
+  };
+
   // Queries disputable liquidations and disputes any that were incorrectly liquidated.
   queryAndDispute = async priceFunction => {
     Logger.debug({
       at: "Disputer",
       message: "Checking for any disputable liquidations"
     });
 
-    // Update the client to get the latest liquidation information.
-    await this.empClient._update();
-
-    // Update the gasEstimator to get the latest gas price data.
-    // If the client has a data point in the last 60 seconds returns immediately.
-    await this.gasEstimator._update();
+    await this.update();
```

Comment:
Looks like this PR has repeated changes from past PRs? Maybe rebase your most recent commit onto master?

Reply:
@mrice32 i forgot to merge master, check now!